libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxVoiceInstructionsPlayer.kt

```diff
@@ -92,9 +92,11 @@ class MapboxVoiceInstructionsPlayer @JvmOverloads constructor(
                 abandonFocus()
             }
 
-            val currentAnnouncement = currentPlayCallback.announcement
-            val currentClientCallback = currentPlayCallback.consumer
-            currentClientCallback.accept(currentAnnouncement)
+            if (currentPlayCallback != null) {
```

Comment:
Can you add the following test to `MapboxVoiceInstructionsPlayerTest`:
```
    @Test
    fun `play is done after clear`() {
        val anyLanguage = Locale.US.language
        val mockedAnnouncement: SpeechAnnouncement = mockk(relaxed = true)
        val mapboxVoiceInstructionsPlayer =
            MapboxVoiceInstructionsPlayer(
                aMockedContext,
                anyLanguage,
                mockedPlayerOptions
            )
        val mockedPlay: SpeechAnnouncement = mockedAnnouncement
        val voiceInstructionsPlayerConsumer =
            mockk<MapboxNavigationConsumer<SpeechAnnouncement>>(relaxed = true)

        val requestSlotCallback = slot<AudioFocusRequestCallback>()
        every {
            mockedAudioFocusDelegate.requestFocus(any(), capture(requestSlotCallback))
        } just Runs
        mapboxVoiceInstructionsPlayer.play(mockedPlay, voiceInstructionsPlayerConsumer)
        mapboxVoiceInstructionsPlayer.clear()

        requestSlotCallback.captured.invoke(true)

        verify(exactly = 0) { voiceInstructionsPlayerConsumer.accept(any()) }
    }
```

Reply:
Sure, done.

Reply:
Thanks. I'd wait for Tomasz though, he's the one who's been working with voice instructions lately.

Reply:
Oh, we's out until after the holidays, let's merge.